### PR TITLE
Hartk board mount for galileo 

### PR DESCRIPTION
**Pull request by majarspeed**
* [X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [X] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [X] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [X] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [X] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [X] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [X] Submitted STL files are oriented and scaled properly for printing.
  * [X] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

### CODE REVIEW

File: printer_mods/Majarspeed/Hartk_Mount_Galileo/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Created folder
```
Comment (KiloQubit) [inline printer_mods/Majarspeed/Hartk_Mount_Galileo/README.md:1]: Please describe the mod in this README, including a link to hartk's board.  Also include the image of the mount that you have in this PR.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod.

Thanks for your mod!  Please see needed changes below.

Please remove spaces from all filenames.

One part per STL - your spacer STL should only contain one spacer, and if users need to print two of them, then have x2 at the end of the file name.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
You deleted the mount STL, now the mod only has the spacer STL.

Please remove the space from your JPG filename.

Please modify your README to describe the mod, and include your JPG image in there.

File: printer_mods/Majarspeed/Hartk_Mount_Galileo/README.md (hunk 1 above)
Comment (majarspeed) [reply printer_mods/Majarspeed/Hartk_Mount_Galileo/README.md:1]: No, I started working on it and got busy and never came back that night to finish it. All the edits should be good now. Created a folder all the spaces should be gone added an image and wrote a very simple description with link to git for the board and a purchase link.

**Review by majarspeed — COMMENTED** [review]
All requested changes should be complete now.

File: printer_mods/Majarspeed/README.md [hunk 2]
```diff
@@ -1 +1,6 @@
 Simple mount for Hartk board for galileo.
+Used to mount the Htark board using the cover screw mouting locations for the galelieo. Uses 1 m3 heat insert and 2 cative M3 nuts.
```
Comment (KiloQubit) [inline printer_mods/Majarspeed/README.md:2]: Replace `Htark` with `Hartk`.

Replace `mouting` with `mounting`.

Replace `galelieo` with `Galileo`

Replace `cative` with `captive`.

File: printer_mods/Majarspeed/README.md [hunk 3]
```diff
@@ -1 +1,6 @@
 Simple mount for Hartk board for galileo.
+Used to mount the Htark board using the cover screw mouting locations for the galelieo. Uses 1 m3 heat insert and 2 cative M3 nuts. 
+Board is avaliable for purchase here https://formosissima.com/products/toolhead-board-for-voron-v2-4 
+Git hub for the board  located here https://github.com/hartk1213/Voron-Hardware/tree/hartk1213/Voron-Hardware/Afterburner_Toolhead_PCB
+
+Thanks to Hartk for making and amazing board for out voron needs.
```
Comment (KiloQubit) [inline printer_mods/Majarspeed/README.md:6]: Replace `and` with `an`.

Replace `out` with `our`.

File: printer_mods/Majarspeed/README.md [hunk 4]
```diff
@@ -1 +1,6 @@
 Simple mount for Hartk board for galileo.
```
Comment (KiloQubit) [inline printer_mods/Majarspeed/README.md:1]: This is the wrong README for all this text.  All this text should go into `printer_mods/Majarspeed/Hartk_Mount_Galileo/README.md`

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the update!

Please rename `Hartk_board_mount_galelieo.stl` to `Hartk_board_mount_galileo.stl`

Please include the images you have in this PR in your README.

File: printer_mods/Majarspeed/README.md (hunk 4 above)
Comment (majarspeed) [reply printer_mods/Majarspeed/README.md]: I have it in both of them.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the updates.

It looks like `Hartk _optional_spacer_X2.stl` still has a space in the filename after `Hartk`.

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**